apps/etterlatte-saksbehandling-ui/client/src/components/tilbakekreving/vurdering/TilbakekrevingVurderingSkjema.tsx

```diff
@@ -170,16 +204,9 @@ export function TilbakekrevingVurderingSkjema({
             </>
           }
         />
-
-        {watch().tilsvar?.tilsvar == JaNei.JA && (
+        {getValues().tilsvar?.tilsvar == JaNei.JA && (
```

Comment:
hva er grunnen til `getValues()` i stedet for `watch()` her? Tror jeg byttet til `watch()` tidligere etter tilbakemeldinger på en annen PR

Reply:
Ok. Jeg bare ser i doken at watch() trigger re-rendering. Kanskje @Watercolours har noe å si om dette?